### ISSUE TITLE
libsoup3 support

### ISSUE THREAD

**Issue by meerfrau** [frafra/frakegps#26]
Hi Francesco,

it would be great if you could [migrate away](https://libsoup.org/libsoup-3.0/migrating-from-libsoup-2.html) from libsoup2:
```
error: failed to run custom build command for `soup-sys v0.10.0`
  --- stderr
  Package libsoup-2.4 was not found in the pkg-config search path.
```
Is frakegps itself using soup-sys or just via https://github.com/tauri-apps/webkit2gtk-rs ?

**Comment by frafra** [comment]
Thanks for filling out this issue :)

Here is the dependency chain: `frakegps -> web-view -> webview-sys -> webkit2gtk-sys -> soup-sys`

It should be enough to do a version bump. I will do it over the weekend.

**Comment by frafra** [comment]
There are no updates for `web-view` or `webview-sys`, so a version bump is not enough. I should use `wry` instead, probably.

**Comment by frafra** [comment]
Fixed in aaefc6e8a9b47f8e8378d4340d73dcdf1d3a3420.
